src/components/accession2/view/Accession2View.tsx

```diff
@@ -151,20 +169,23 @@ export default function Accession2View(): JSX.Element {
         setAge(strings.formatString(strings.AGE_VALUE_MONTHS, accessionAge) as string);
       }
     }
-  }, [accession, activeLocale]);
+  }, [accession, activeLocale, seedBankTimeZone]);
 
   useEffect(() => {
     if (activeLocale) {
       if (accession?.dryingEndDate) {
-        // MomentJS has its own version of a "gibberish" locale, but with a different name.
-        const momentLocale = activeLocale === 'gx' ? 'x-pseudo' : activeLocale;
-        const dryingMoment = moment(accession.dryingEndDate).locale(momentLocale);
-        setDryingRelativeDate(dryingMoment.fromNow());
+        // luxon has no pseudo locales, use Korean for gibberish.
```

Comment:
@sgrimm i had to move away from moment for this implementation - since it's relative time display uses client browser time. Luxon does not have a pseudo locale equivalent.. using Korean for now.